tests/safeds_stubgen/stubs_generator/__snapshots__/test_generate_stubs/TestStubFileGeneration.test_stub_creation[_module_3].sdsstub

```diff
@@ -1,4 +1,4 @@
-@PythonModule("tests.data.various_modules_package.file_creation._module_3")
-package tests.data.variousModulesPackage.fileCreation.Module3
+@PythonModule("various_modules_package")
+package variousModulesPackage
```

Comment:
The removal of `file_creation._module_3` is correct, but the prefix `tests.data` should remain in this case.

Reply:
The same applies to the other tests here.